pom.xml

```diff
@@ -469,6 +475,91 @@
+                <!-- Apache License 2.0 https://spdx.org/licenses/Apache-2.0.html -->
+                <licenseMerge>Apache-2.0|Apache 2.0</licenseMerge>
+                <licenseMerge>Apache-2.0|Apache License, Version 2.0</licenseMerge>
+                <licenseMerge>Apache-2.0|Apache Software License, Version 2.0</licenseMerge>
+                <licenseMerge>Apache-2.0|The Apache Software License, Version 2.0</licenseMerge>
+                <licenseMerge>Apache-2.0|The Apache License, Version 2.0</licenseMerge>
+                <licenseMerge>Apache-2.0|Apache 2.0 License</licenseMerge>
+                <licenseMerge>Apache-2.0|Apache License 2.0</licenseMerge>
+                <licenseMerge>Apache-2.0|ASL, version 2</licenseMerge>
+                <licenseMerge>Apache-2.0|Apache License V2.0</licenseMerge>
+                <licenseMerge>Apache-2.0|Apache License, version 2.0</licenseMerge>
+                <licenseMerge>Apache-2.0|Apache License, 2.0</licenseMerge>
+                <licenseMerge>Apache-2.0|Apache 2</licenseMerge>
+                <licenseMerge>Apache-2.0|Apache Software License, version 2.0</licenseMerge>
+                <licenseMerge>Apache-2.0|Apache License Version 2.0</licenseMerge>
+                <!-- Eclipse Public License 1.0 https://spdx.org/licenses/EPL-1.0.html -->
+                <licenseMerge>EPL-1.0|Eclipse Public License - v 1.0</licenseMerge>
+                <licenseMerge>EPL-1.0|Eclipse Public License 1.0</licenseMerge>
+                <!-- Eclipse Public License 2.0 https://spdx.org/licenses/EPL-2.0.html -->
+                <licenseMerge>EPL-2.0|Eclipse Public License - v 2.0</licenseMerge>
+                <licenseMerge>EPL-2.0|EPL 2.0</licenseMerge>
+                <!-- BSD 2-Clause "Simplified" License https://spdx.org/licenses/BSD-2-Clause.html -->
+                <licenseMerge>BSD-2-Clause|BSD 2-Clause</licenseMerge>
+                <licenseMerge>BSD-2-Clause|BSD 2-Clause License</licenseMerge>
+                <licenseMerge>BSD-2-Clause|The 2-Clause BSD License</licenseMerge>
+                <licenseMerge>BSD-2-Clause|New BSD License</licenseMerge>
+                <!-- GNU Lesser General Public License v2.1 only https://spdx.org/licenses/LGPL-2.1.html -->
+                <licenseMerge>LGPL-2.1|GNU Lesser General Public License</licenseMerge>
+                <licenseMerge>LGPL-2.1|GNU Lesser General Public License, Version 2.1</licenseMerge>
+                <licenseMerge>LGPL-2.1|GNU Lesser Public License</licenseMerge>
+                <licenseMerge>LGPL-2.1|LGPL, version 2.1</licenseMerge>
+                <licenseMerge>LGPL-2.1|LGPL 2.1</licenseMerge>
+                <licenseMerge>LGPL-2.1|LGPL</licenseMerge>
+                <!-- GNU Library General Public License v2 only https://spdx.org/licenses/LGPL-2.0.html -->
+                <licenseMerge>LGPL-2.0|GNU General Public Library</licenseMerge>
```

Comment:
This is wrong. stax-api declares

```
      <name>GNU General Public Library</name>
      <url>http://www.gnu.org/licenses/gpl.txt</url>
```

This is not LGPL. There is no version information.

Reply:
I've removed all licenseMerges.

Reply:
Do you mean that license merging approach unneeded at all here? Or there is some kind of inaccuracy with determining which  license to what identifier belong for.

Reply:
stax-api uses GPL and not LGPL. The merge for stax-api was wrong.
License merging is not needed.
If you want to spent time on this it must be correct.

Reply:
What are the benefits of license merging? Is it only that fewer license files are included in the jar?